addons/controllers/calicoconfig_controller_test.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const testCluster = "test-cluster-calico"
 
-var _ = Describe("CalicoConfig Reconciler", func() {
+var _ = FDescribe("CalicoConfig Reconciler", func() {
```

Comment:
blocking: we might want to change back to `Describe` before merge.

Reply:
I don't plan to check this in. This is to only facilitate testing for time being